isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-service/inhalt.adoc

```diff
@@ -115,27 +115,14 @@ Gibt es allerdings Anforderungen, aus denen heraus die Service-Fassade eine Tran
 Die Aufgabe fällt der Service-Fassade zu, weil es wichtig ist, dass die Fehlerbehandlung auf jeden Fall die Transaktion umschließt.
 Nur so ist gewährleistet, dass auch Fehler, die beim Commit entstehen, von der Fehlerbehandlung erfasst werden.
 
-Die Transaktionssteuerung wird an der jeweiligen Service-Methode angesetzt.
-Für das obige Beispiel verdeutlicht dies <<listing-service-fassade-tx>>.
+Die Transaktionalität einzelner Service-Methoden wird über Annotationen an diesen Methoden kofiguriert:
+
 .Transaktionssteuerung in der Service-Fassade
```

Comment:
Zeile 120-123 und Zeile 125 müssen noch entfernt werden, da sie zu dem entfernten Beispiel gehören.

Reply:
Die Zeilen hab ich jetzt entfernt. 
Ihr Inhalt war genau das was jetzt verkürzt in Klammern steht:

@Transactional(rollbackFor = Throwable.class, propagation = Propagation.REQUIRED)